src/components/App.tsx

```diff
@@ -1468,19 +1463,34 @@ export class App extends React.Component<any, AppState> {
       this.state,
     );
     if (selectedElements.length === 1 && !isOverScrollBar) {
-      const resizeElement = getElementWithResizeHandler(
+      const elementWithResizeHandler = getElementWithResizeHandler(
```

Comment:
Can be a better name. But, eventually we could refactor code here to simplify more.